### PR TITLE
MCLOUD-5709: Add DB access to the CLI

### PR DESCRIPTION
**Pull request by G-Arvind**
### Description

**[MCLOUD-5709:](https://jira.corp.magento.com/browse/MCLOUD-5709)** For repetitive tasks during development, It is more convenient to access the DB from CLI, so I have added a CLI method for command-line access to the DB.

### Fixed Issues (if relevant)

1. magento/magento-cloud-docker#180: Add DB access to the CLI

### Manual testing scenarios

1. Run ``` ./bin/magento-docker ece-db``` to access DB console.

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages

releasenotes
Added the `bin/magento-docker ece-db` command to connect to the database container

### CODE REVIEW

File: dist/bin/magento-docker [hunk 1]
```diff
@@ -16,6 +16,7 @@ USAGE="Magento Cloud Docker
   ece-deploy        run deploy hooks
   ece-post-deploy   run post-deploy hooks
   ece-redeploy      run build, deploy, and post-deploy hooks
+  mysql             access to mysql DB
```
Comment (shiftedreality) [inline dist/bin/magento-docker]: It's better to name it ece-db, to be more generic and support both MariaDB and Mysql

File: dist/bin/magento-docker [hunk 2]
```diff
@@ -62,6 +63,9 @@ case "$1" in
         docker-compose run --rm deploy cloud-deploy
         docker-compose run --rm deploy cloud-post-deploy
         ;;
+    mysql)
```
Comment (shiftedreality) [inline dist/bin/magento-docker]: `ece-db`

File: dist/bin/magento-docker [hunk 3]
```diff
@@ -62,6 +63,9 @@ case "$1" in
...
+        docker-compose exec db sh -c 'mysql -u $MYSQL_USER -p$MYSQL_PASSWORD $MYSQL_DATABASE "$@"'
```
Comment (shiftedreality) [inline dist/bin/magento-docker:67]: Please use `run --rm` instead

**Review by shiftedreality — CHANGES_REQUESTED** [review]

File: dist/bin/magento-docker (hunk 3 above)
Comment (aepod) [reply dist/bin/magento-docker:67]: This creates a new container, which will not have access to the DB, which is why he was using exec, not `run --rm`

File: dist/bin/magento-docker (hunk 1 above)
Comment (G-Arvind) [reply dist/bin/magento-docker]: Changed to ece-db

**Review by oshmyheliuk — APPROVED** [review]

File: dist/bin/magento-docker (hunk 3 above)
Comment (shiftedreality) [reply dist/bin/magento-docker:67]: Cool, makes sense them

**Review by shiftedreality — APPROVED** [review]

**Comment by andriyShevtsov** [comment]
QA approved

**Pull request merged**